### ISSUE TITLE
Can't access Firefly-III after upgrading to 4.7.12 from 4.7.10

### ISSUE THREAD

**Issue by Ryonez** [firefly-iii/firefly-iii#2095]
**Bug description**
I am running Firefly III version 4.7.12, and my problem is:

**Steps to reproduce**

Upgrade from 4.7.10 from 4.7.12
Navigate to the page and be greeted with a "Be right back." message.

**Expected behavior**

For firefly to be accessible.

**Extra info**

Runnind via docker on unRaid, using postgres and ldap 

**Bonus points**

![image](https://user-images.githubusercontent.com/16157130/52910164-4dcbf080-32f8-11e9-9f84-a92ba8d579e5.png)

Logs:
```
Now in entrypoint.sh for Firefly III
Architecture: x86_64
CPU op-mode(s): 32-bit, 64-bit
Byte Order: Little Endian
CPU(s): 8
On-line CPU(s) list: 0-7
Thread(s) per core: 2
Core(s) per socket: 4
Socket(s): 1
NUMA node(s): 1
Vendor ID: GenuineIntel
CPU family: 6
Model: 26
Model name: Intel(R) Xeon(R) CPU W3550 @ 3.07GHz
Stepping: 5
CPU MHz: 3080.948
CPU max MHz: 3060.0000
CPU min MHz: 1596.0000
BogoMIPS: 6133.87

Virtualization: VT-x
L1d cache: 32K
L1i cache: 32K
L2 cache: 256K
L3 cache: 8192K
NUMA node0 CPU(s): 0-7
Flags: fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx rdtscp lm constant_tsc arch_perfmon pebs bts rep_good nopl xtopology nonstop_tsc cpuid aperfmperf pni dtes64 monitor ds_cpl vmx est tm2 ssse3 cx16 xtpr pdcm dca sse4_1 sse4_2 popcnt lahf_lm pti ssbd ibrs ibpb stibp tpr_shadow vnmi flexpriority ept vpid dtherm ida flush_l1d
Making directories...
Touch DB file (if SQLlite)...
Run chown on /var/www/firefly-iii/storage...
Run chmod on /var/www/firefly-iii/storage...
Remove log file...
Map environment variables on .env file...
Dump auto load...
Generating optimized autoload files> Illuminate\Foundation\ComposerScripts::postAutoloadDump
Generated optimized autoload files containing 4717 classes
Discover packages...
Discovered Package: adldap2/adldap2-laravel
Discovered Package: danhunsaker/laravel-flysystem-others
Discovered Package: davejamesmiller/laravel-breadcrumbs
Discovered Package: fideloper/proxy
Discovered Package: laravel/nexmo-notification-channel
Discovered Package: laravel/passport
Discovered Package: laravel/slack-notification-channel
Discovered Package: laravelcollective/html
Discovered Package: nesbot/carbon
Discovered Package: pragmarx/google2fa-laravel
Discovered Package: rcrowe/twigbridge
Package manifest generated successfully.
Run various artisan commands...
Migrating: 2019_02_05_055516_changes_for_v4711
Migrated: 2019_02_05_055516_changes_for_v4711
Migrating: 2019_02_11_170529_changes_for_v4712
Migrated: 2019_02_11_170529_changes_for_v4712
Seeding: AccountTypeSeeder
Seeding: TransactionCurrencySeeder
Seeding: TransactionTypeSeeder
Seeding: PermissionSeeder
Seeding: LinkTypeSeeder
Seeding: ConfigSeeder
Database seeding completed successfully.
Going to decrypt the database.
No decryption required for table "accounts".
No decryption required for table "attachments".
No decryption required for table "bills".
No decryption required for table "budgets".
No decryption required for table "categories".
No decryption required for table "piggy_banks".
No decryption required for table "preferences".
No decryption required for table "tags".
No decryption required for table "transaction_journals".
No decryption required for table "transactions".
No decryption required for table "journal_links".
Done!

In CreatesUserProviders.php line 42:

Authentication user provider [adldap] is not defined.


{REDACTED}
Amount integrity OK for user #3
Amount integrity OK for user #2
Amount integrity OK for user #1
No orphaned transactions!
No orphaned journals!
All link types OK!
All access tokens OK!
Amount integrity OK!
Encryption keys generated successfully.
Personal access client created successfully.
Client ID: 71
Client secret: {REDACTED}
Password grant client created successfully.
Client ID: 72
Client secret: {REDACTED}
Application cache cleared!
+------------------------------------------------------------------------------+
| |
| Thank you for installing Firefly III, v4.7.12! |
| |
| |
+------------------------------------------------------------------------------+
Go!
[Sun Feb 17 20:59:47.340177 2019] [mpm_prefork:notice] [pid 1] AH00163: Apache/2.4.25 (Debian) PHP/7.2.15 OpenSSL/1.0.2q configured -- resuming normal operations
[Sun Feb 17 20:59:47.340222 2019] [core:notice] [pid 1] AH00094: Command line: 'apache2 -D FOREGROUND'
172.17.0.1 - - [17/Feb/2019:20:59:47 +1300] "GET /debug HTTP/1.1" 500 2767 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.109 Safari/537.36"
172.17.0.1 - - [17/Feb/2019:20:59:48 +1300] "GET /fonts/lato-100.woff2 HTTP/1.1" 404 3438 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.109 Safari/537.36"
172.17.0.1 - - [17/Feb/2019:20:59:48 +1300] "GET /fonts/lato-100.woff HTTP/1.1" 404 3438 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.109 Safari/537.36"
172.17.0.1 - - [17/Feb/2019:20:59:48 +1300] "GET /favicon.ico HTTP/1.1" 200 12261 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.109 Safari/537.36"
172.17.0.1 - - [17/Feb/2019:20:59:50 +1300] "GET /debug HTTP/1.1" 500 2779 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.109 Safari/537.36"
172.17.0.1 - - [17/Feb/2019:20:59:50 +1300] "GET /fonts/lato-100.woff2 HTTP/1.1" 404 3438 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.109 Safari/537.36"
172.17.0.1 - - [17/Feb/2019:20:59:50 +1300] "GET /fonts/lato-100.woff HTTP/1.1" 404 3438 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.109 Safari/537.36"
172.17.0.1 - - [17/Feb/2019:20:59:50 +1300] "GET /favicon.ico HTTP/1.1" 200 12261 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.109 Safari/537.36"
172.17.0.1 - - [17/Feb/2019:20:59:55 +1300] "GET / HTTP/1.1" 500 2773 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.109 Safari/537.36"
172.17.0.1 - - [17/Feb/2019:20:59:55 +1300] "GET /fonts/lato-100.woff2 HTTP/1.1" 404 3438 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.109 Safari/537.36"
172.17.0.1 - - [17/Feb/2019:20:59:55 +1300] "GET /fonts/lato-100.woff HTTP/1.1" 404 3438 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.109 Safari/537.36"
172.17.0.1 - - [17/Feb/2019:21:00:38 +1300] "GET / HTTP/1.1" 500 2775 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.109 Safari/537.36"
172.17.0.1 - - [17/Feb/2019:21:00:39 +1300] "GET /fonts/lato-100.woff2 HTTP/1.1" 404 3438 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.109 Safari/537.36"
172.17.0.1 - - [17/Feb/2019:21:00:39 +1300] "GET /fonts/lato-100.woff HTTP/1.1" 404 3438 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.109 Safari/537.36"
172.17.0.1 - - [17/Feb/2019:21:00:40 +1300] "GET /favicon.ico HTTP/1.1" 200 12261 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.109 Safari/537.36"
172.17.0.1 - - [17/Feb/2019:21:00:40 +1300] "GET /favicon.ico HTTP/1.1" 200 12261 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.109 Safari/537.36"
172.17.0.1 - - [17/Feb/2019:21:00:50 +1300] "GET / HTTP/1.1" 500 2771 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.109 Safari/537.36"
172.17.0.1 - - [17/Feb/2019:21:00:50 +1300] "GET /fonts/lato-100.woff2 HTTP/1.1" 404 3438 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.109 Safari/537.36"
172.17.0.1 - - [17/Feb/2019:21:00:50 +1300] "GET /fonts/lato-100.woff HTTP/1.1" 404 3438 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.109 Safari/537.36"
172.17.0.1 - - [17/Feb/2019:21:00:50 +1300] "GET /favicon.ico HTTP/1.1" 200 12261 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.109 Safari/537.36"
172.17.0.1 - - [17/Feb/2019:21:09:00 +1300] "GET /debug HTTP/1.1" 500 2771 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.109 Safari/537.36"
172.17.0.1 - - [17/Feb/2019:21:09:00 +1300] "GET /fonts/lato-100.woff2 HTTP/1.1" 404 3438 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.109 Safari/537.36"
172.17.0.1 - - [17/Feb/2019:21:09:00 +1300] "GET /fonts/lato-100.woff HTTP/1.1" 404 3438 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.109 Safari/537.36"```

**Comment by JC5** [comment]
As mentioned in the release notes, to keep using LDAP, set the `LOGIN_PROVIDER` to `ldap`.

**Comment by Ryonez** [comment]
I had a suspicion `Authentication user provider [adldap] is not defined.` may have been related, and was rolling back to compare logs as I knew ldap worked before.

I didn't realize other fixes/changes where deployed with 4.7.12. I'll make sure to check the release info next time, thank you for the help.
